src/App.js

```diff
@@ -1,20 +1,26 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Route,
+  Routes,
+} from 'react-router-dom';
 import './index.css';
-import Calculator from './components/calculator';
+import Navbar from './components/Navbar';
+import Calculator from './components/Calculator';
```

Comment:
## Optional
- [ ] Please change this import the file is `calculator` not `Calculator` it was causing the server not to run

Reply:
- [ ] Remember to add the live version in the README